rental.rb

```diff
@@ -0,0 +1,11 @@
+class Rental
+  attr_accessor :date
```

Comment:
- [ ] There should be `attr_accessor :date, :book, :person`. KIndly add it.

Reply:
Well resolved 👍🏽